.env.example

```diff
@@ -0,0 +1,2 @@
+OPENAI_API_KEY='Your Open API Key, e.g. sk-xxx '
```

Comment:
This is an example of something called an `.env` file. These files are used to store secrets that you don't want to expose to the public.

You do not commit an `.env` file to Github. This file is usually handled in wherever you deploy your code like Heroku, GH, AWS, Vercel, etc.

Reply:
When you clone the repo, you should create a `.env` file and fill it in following the example here.